x/auth/client/context/context.go

```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/spf13/viper"
 )
 
-// TxContext implements a transaction context created in SDK modules.
-type TxContext struct {
+// TxBuilder implements a transaction context created in SDK modules.
```

Comment:
Since this is a `builder`, I think the path should be renamed: `x/auth/client/builder/builder.go`

Reply:
done, txbuilder since builder is probably the wrong supergroup to create a module over.